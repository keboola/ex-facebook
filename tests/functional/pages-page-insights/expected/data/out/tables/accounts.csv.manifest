{"primary_key":["id"],"columns":["id","name","category","category_list","tasks"]}