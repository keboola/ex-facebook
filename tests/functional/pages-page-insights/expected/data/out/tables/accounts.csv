"133353749860945","Test extractor page","Information technology company","[{""id"":""1130035050388269"",""name"":""Information technology company""}]","[""ADVERTISE"",""ANALYZE"",""CREATE_CONTENT"",""MESSAGING"",""MODERATE"",""MANAGE""]"
